docs/reference/stable-patch-format.rst

```diff
@@ -0,0 +1,366 @@
+   **all** patches against released kernels **must** have two "Acked-by"
+   replies by members of the Ubuntu Kernel Team.
+
+   Example::
+
+     Signed-off-by: Adam Jackson <ajax@redhat.com>
+     Signed-off-by: Eric Anholt <eric@anholt.net>
+     Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
+     (cherry picked from commit d4e0018e3e4dd685af25d300fd26a0d5a984482e 2.6.34.y)
+     Signed-off-by: Manoj Iyer <manoj.iyer@canonical.com>
+     Acked-by: Tim Gardner <tim.gardner@canonical.com>
+     Acked-by: Brad Figg <brad.figg@canonical.com>
+     Acked-by: Steve Conklin <sconklin@canonical.com>
+
+#. Every patch **must** display the provenance of the patch. We want to
+   preserve where the patch came from, who signed off on it, who ack'd it,
+   whether it was cherry-picked from upstream and applied cleanly or not and
+   who finally applied it to an official Ubuntu source tree. Cherry-picked from
+   vs backported from:
+
+   If the patch required changes (it did not apply cleanly), 'backported from
+   commit <sha1>' has to be used i.e::
+
+     (backported from commit <sha1> <upstream repo name>)
+
+   Please note that the opening and closing parenthesis are required. There
+   must be a brief explanation immediately after the "(backported from ...)"
+   block, between square brackets, with the name of the person who introduced
+   the change::
+
+     (backported from commit <sha1> <upstream repo name>)
+     [roxanan: Had to adjust the context due to missing commit <sha1>]
+
+   If the patch is a simple cherry-pick from an upstream repo and it applies
+   cleanly, that **must** also be spelled out in the provenance section in the
+   form by using 'cherry-picked from'::
+
+     (cherry picked from commit <sha1> <upstream repo name>)
+
+   The opening and closing parenthesis are required here as well.
+
+   .. note::
+
+      "<upstream repo name>" should be omitted if the patch comes from the
+      mainline tree.
+
+   Example::
+
+     Signed-off-by: Adam Jackson <ajax@redhat.com>
+     Signed-off-by: Eric Anholt <eric@anholt.net>
+     Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
+     (cherry picked from commit d4e0018e3e4dd685af25d300fd26a0d5a984482e 2.6.34.y)
+     Signed-off-by: Manoj Iyer <manoj.iyer@canonical.com>
+     Acked-by: Tim Gardner <tim.gardner@canonical.com>
+     Acked-by: Brad Figg <brad.figg@canonical.com>
+     Acked-by: Steve Conklin <sconklin@canonical.com>
+
+   .. _comment-body-cve:
+
+#. Every **CVE** patch **must** contain a line at the beginning of the commit
+   message that specifies the CVE number(s) related to the patch. This must be
+   the first part of the body of the comment. There is the comment subject
+   line, a blank line, the CVE number, a blank line and then the rest of the
+   comment body.  A "BugLink" is optional for CVE patches.
+
+   Example::
+
+     Subject: [SRU B/D] UBUNTU: SAUCE: nbd_genl_status: null check for nla_nest_start
```

Comment:
This is just copied from the old doc. It should be [SRU][B/D] now. I'll fix this.